src/components/structures/IndicatorScrollbar.js

```diff
@@ -131,17 +131,29 @@ export default class IndicatorScrollbar extends React.Component {
             // the harshness of the scroll behaviour. Should be a value between 0 and 1.
             const yRetention = 1.0;
 
-            // Check for trackpad users every so often to avoid boosting their scroll.
-            // See https://github.com/vector-im/riot-web/issues/10005
+            // whenever see horizontal scrolling, assume the user is on a trackpad
+            // for at least the next 15 minutes.
             const now = new Date().getTime();
-            if (now >= this._checkAgainForTrackpad) {
-                this._likelyTrackpadUser = Math.abs(e.deltaX) > 0;
+            if (Math.abs(e.deltaX) > 0) {
+                this._likelyTrackpadUser = true;
                 this._checkAgainForTrackpad = now + (15 * 60 * 1000); // 15min
             }
+            else {
+                // if we haven't seen any horizontal scrolling for >15 minutes, assume
```

Comment:
we should bump this down to a minute if we're not sampling.

Reply:
sgtm